utils/fetcher.ts

```diff
@@ -0,0 +1,48 @@
+type HTTPMethod =
+  | 'GET'
+  | 'HEAD'
+  | 'POST'
+  | 'PUT'
+  | 'DELETE'
+  | 'CONNECT'
+  | 'OPTIONS'
+  | 'TRACE'
```

Comment:
Czemu wcześniej metody http były małymi literami a tu są wielkimi? Potrzebujemy w ogóle na to dwóch osobnych typów?

Reply:
Nie potrzebujemy, wywalę jeden typ.